classifier/logistic_regression.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+
+def sigmoid(x):
+    return 1 / (1 + np.exp(-x))
+
+class LogisticRegression:
+    
+    def __init__(self, learning_rate=0.001, n_iters=1000):
+        self.learning_rate = learning_rate
+        self.n_iters = n_iters
+        self.classifiers = {}
+
+    def fit(self, X, y):
+        unique_classes = np.unique(y)
+        for cls in unique_classes:
+            binary_y = np.where(y == cls, 1, 0)
+            self.classifiers[cls] = self.train_classifier(X, binary_y)
+
+    def train_classifier(self, X, y):
```

Comment:
Make this field private

`__train_classifier(self, X, y):`

Reply:
to take